gio/ginetaddress.c

```diff
@@ -711,13 +755,19 @@ g_inet_address_get_is_any (GInetAddress *address)
   g_return_val_if_fail (G_IS_INET_ADDRESS (address), FALSE);
 
   if (address->priv->family == AF_INET)
-    {
-      guint32 addr4 = g_ntohl (address->priv->addr.ipv4.s_addr);
+  {
+    guint32 addr4 = g_ntohl (address->priv->addr.ipv4.s_addr);
 
-      return addr4 == INADDR_ANY;
-    }
+    return addr4 == INADDR_ANY;
+  }
   else
+  {
+#ifdef HAVE_IPV6
     return IN6_IS_ADDR_UNSPECIFIED (&address->priv->addr.ipv6);
+#else
+    return FALSE;
+#endif
+  }
```

Comment:
Block indentation is off.

Reply:
(Would be good to avoid adding blocks when not needed though, to make the patch smaller.)

Reply:
Should I remove the block on line 764? It looks like this

    {
#ifdef HAVE_IPV6
      return IN6_IS_ADDR_UNSPECIFIED (&address->priv->addr.ipv6);
#else
      return FALSE;
#endif
    }

I thought we wanted to keep the blocks unless the statement was only a single line. Although the pre-processor will reduce this to a single line, should the braces be left here to make it easier to read?

Reply:
Normally we would use a block, but it will make this patch larger and harder to maintain. (In case upstream isn't interested in this improvement.)